bin/submarine.js

```diff
@@ -8,10 +7,11 @@ var options = {
   output_dir: args._[1],
   header: args.header,
   footer: args.footer,
-  template: args.template
+  template: args.template,
+  assets: args.assets
```

Comment:
We are taking in a directory, so lets say `assets_dir` to match `input_dir` and `output_dir` above.